content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,157 @@
+
+Let's take a look at the control signal (also called "[timecode](https://en.wikipedia.org/wiki/Timecode) signal").
+If you open the WAV file with an audio editor like Audacity, it looks like this:
+
+![Serato CD Timecode Signal in Audacity]({static}/images/news/timecode-signal.png)
+
+As you can see, the timecode signal is a stereo signal.
+The left channel is on top and the right channel is on the bottom.
+Both channels look very similar, but the right channel is shifted a little.
+
+## Detecting the Playback Velocity
+
+Let's focus on the left channel for now.
+The wave basically looks similar to a sine wave that you may remember from your high school math classes.
+You can divide that signal into "cycles", where each cycle starts from zero and goes into a positive part (above the black line) followed by a negative part (below the black line) and then repeats:
+
+![Cycle of left channel in timecode signal]({static}/images/news/timecode-signal-cycle.png)
+
+Each cycle has the same length and if we count the number of cycles in a single second of audio, there are 1000 of them.
+Hence, we now know that the signal has a frequency of 1000 Hz (Hz = 1/s, so it's basically a fancy way saying "per second").
+
+With that information, the DVS system can detect how fast the record is playing by checking how many cycles there are in a second.
+If there are 2000 cycles per second, it's playing a double speed, at 500 cycles per second the record is playing at half speed, and so on.
+
+But how does the computer know how many cycles there are in a second?
+We can't actually wait for a second, because the delay (also called latency) would be very noticeable and make the system unsuitable for scratching.
+
+## Analog vs. Digital Audio
+
+It's important to know that in contrast to analog audio like vinyl or music cassettes, the computer doesn't actually work with the continuous wave that we see in the screenshot.
+
+![Analog Signal]({static}/images/news/signal_analog.svg)
+
+Instead, [digital signals](https://en.wikipedia.org/wiki/Digital_signal_%28signal_processing%29) work with measurements (called "samples") that indicate the value of the wave position at specific points in time.
+How often these measurements are taken is determined by the sample rate (or sampling rate).
+For example, Audio CDs use a sample rate of 44100 Hz (i.e. 44100 measurements per second, or 1 measurement every ~22.68 microseconds).
+Such a signal is called "time-discrete":
+
+![Time-Discrete Signal]({static}/images/news/signal_time_discrete.svg)
+
+The reason for this is that a time-continuous signal has infinite resolution and would need infinite memory space to save, which is impossible - and also unnecessary, because as long as you take enough samples per second, you can [restore the original signal](https://en.wikipedia.org/wiki/Nyquist%E2%80%93Shannon_sampling_theorem)  without any loss of information.
```

Comment:
```suggestion
The reason for this is that a time-continuous signal has infinite resolution and would need infinite memory space to save, which is impossible - and also unnecessary, because as long as you take enough samples per second, you can [restore the original signal](https://en.wikipedia.org/wiki/Nyquist%E2%80%93Shannon_sampling_theorem) without any loss of information.
```
unnecessary space